domino/domino.py

```diff
@@ -499,6 +499,12 @@ def throw_if_information_invalid(key: str, info: dict) -> bool:
                     f"'maxWorkerCount' is not supported in Domino {self._version}."
                 )
 
+            if "masterHardwareTierId" in compute_cluster_properties:
```

Comment:
Thanks for the feedback, you're right that I don't need to check for the existence of the key. Although, after review it looks like we still need to check for the `masterHardwareTierId` as it can be optional when using an `MPI` cluster type.

See lines `450-460`:
https://github.com/dominodatalab/python-domino/blob/75ee9c47bd29894c4bac24056cad66387932f280/domino/domino.py#L443-L460

I've updated the code so that `workerHardwareTier` doesn't have the existence check.